packages/delivery-xml-http-request/delivery.js

```diff
@@ -28,6 +28,7 @@ module.exports = (client, win = window) => ({
       req.setRequestHeader('Bugsnag-Api-Key', event.apiKey || client._config.apiKey)
       req.setRequestHeader('Bugsnag-Payload-Version', '4')
       req.setRequestHeader('Bugsnag-Sent-At', (new Date()).toISOString())
+      req.setRequestHeader('Access-Control-Max-Age', '86400')
```

Comment:
the end to end tests all seem to be failing now - could you try making this an integer [as per the docs](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Access-Control-Max-Age)